README.md

```diff
@@ -21,18 +21,35 @@ For a more in-depth overview of openCryptoki, please refer to manual
 Does not work with TPM version 2.0. We plan to remove the TPM token in a future 
 openCryptoki release or version.
 
+AIX only supports the CCA and software tokens, and both are enabled in a default build configuration. Other tokens are unsupported and cannot be force-enabled through the `configure` utility.
+
 ## REQUIREMENTS:
 
+### Common
+Building opencryptoki needs the following utilities.
+- flex
+- bison
+- make
+- autoconf
+- automake
+- pkg-config
+- libtool
+- m4
+- openldap-devel
+
+These libraries are usually provided by your platform's package management utilities. On AIX, they must be installed from the AIX Toolbox repositories.
+
+### Tokens
 - IBM ICA - requires libica library version 3.3.0 or higher for accessing ICA
 hardware crypto on IBM zSeries.
 
-- IBM CCA - requires IBM XCrypto CEX3C card (or higher) and the CEX3C host
-libraries and tools version 4.1 (or higher).
+- IBM CCA - requires a IBM XCrypto CEX7S card (or newer) and the CEX7S host
```

Comment:
Well, it is not entirely true that it needs a CEX7S card, at least not on Linux on IBM Z.
What we DO need is the CCA host library of version 7.1 or newer, but the card level is independent of that (at least for IBM Z) and can also be a CEX3. 

Note: On Linux on IBM Z, CEXnC denotes a Crypto express of level 'n' in CCA mode. For EP11 it would be a CEXnP, and in accelerator mode it would be a CEXnA. CEXnS denotes the card in general, independent of the mode it is configured. This might be different in AIX / LoP / LoX86 though.... I guess there we only have adapters in CCA mode.

So maybe we need to rephrase this sentence a bit. Also remove the X from XCrypto....

Reply:
Oh that's allowed on LoZ?

If I remember correctly, I was told the newer releases of CCA (at least 7.x) are not backward-compatible with older cards on AIX. I _assumed_ the support cycle would be similar on other platforms as well - why would there be a disconnect across platforms?

I'll change this sentence accordingly.